.github/workflows/test_with_container_build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           - "opensuse_leap.Dockerfile"
           - "redhat.Dockerfile"
           - "ubuntu.Dockerfile"
-    name: Install Mondoo inside containers to test new releases
+    name: Install Mondoo
```

Comment:
The workflow name already states the container part, making the overview for the matrix jobs in the GitHub UI readable.